petclinic-frontend/src/pages/Home/Home.tsx

```diff
@@ -0,0 +1,11 @@
+import { NavBar } from '@/layouts/AppNavBar.tsx';
+
+// Haitham: created the home page layout
```

Comment:
Comment code like this should not be left in PRs.

Reply:
The comments have been removed.